app/columns.tsx

```diff
@@ -242,6 +242,15 @@ export const fxmsMessageColumns: ColumnDef<FxMSMessageInfo>[] = [
       }
       return <></>;
     },
+    filterFn: (row, columnId, filterValue) => {
+      if (filterValue) {
+        return (
+          (row.original.impressions && row.original.impressions > 1000) ||
```

Comment:
Any thoughts on 1000 impressions being the threshold? I'm open for any suggestions

Reply:
1000 seems like a reasonable place to start.  Could make it an environment variable so it's easy to play with...